scss/bitstyles/ui/extending-bitstyles.stories.mdx

```diff
@@ -0,0 +1,70 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Extending bitstyles" />
+
+# Extending Bitstyles
+
+## Adding your own components
+
+When adding your own custom component in CSS (not a component built using the utility classes), the point at which you import it is important, because CSS is a source order-dependent language.
+
+There are three main layers you need to be aware of:
+
+1. Base styles
+1. Components
+1. Utility classes
```

Comment:
```suggestion
1. Base styles
2. Components
3. Utility classes
```

Reply:
This is deliberate! :D When MD is compiled (or whatever), the list items are numbered correctly. It’s to avoid needing to painfully update numbers when you insert an item, and it’s great (thanks to @klappradla for that 💎)